[2025-05-29T00:01:01.674+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:01:01.678+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:01:01.678+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-05-29T00:01:01.678+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-05-29T00:01:01.678+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-05-29T00:01:01.682+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): store_user> on 2025-05-28 00:00:00+00:00
[2025-05-29T00:01:01.684+0000] {standard_task_runner.py:55} INFO - Started process 24997 to run task
[2025-05-29T00:01:01.686+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2025-05-28T00:00:00+00:00', '--job-id', '620', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpoefrtb_9']
[2025-05-29T00:01:01.687+0000] {standard_task_runner.py:83} INFO - Job 620: Subtask store_user
[2025-05-29T00:01:01.709+0000] {task_command.py:388} INFO - Running <TaskInstance: user_processing.store_user scheduled__2025-05-28T00:00:00+00:00 [running]> on host 74c05f868d26
[2025-05-29T00:01:01.732+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2025-05-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-28T00:00:00+00:00
[2025-05-29T00:01:01.733+0000] {postgres.py:158} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2025-05-29T00:01:01.735+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2025-05-29T00:01:01.737+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-05-29T00:01:01.740+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20250528T000000, start_date=20250529T000101, end_date=20250529T000101
[2025-05-29T00:01:01.784+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-05-29T00:01:01.791+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
